### ISSUE TITLE
Add apple app site association file to source

### ISSUE THREAD

**Issue by Kloenk** [matrix-org/matrix.to#220]
**Is your feature request related to a problem? Please describe.**
Apple requires site association for apps on iOS and open url support for Mac apps

**Describe the solution you'd like**
Add the [/.well-known/apple-app-site-association](https://matrix.to/.well-known/apple-app-site-association) file to the source to make deployment easier and add to make it easier to add custom apps support for matrix.to

**Describe alternatives you've considered**
Patching all clients to not use matrix.to, but something else

**Comment by bwindels** [comment]
The file you link to actually exists ... not sure what is left to do here? Do you want to add an app association for another app than Element iOS?

**Comment by Kloenk** [comment]
Yes, I would like to do something like a pr to it. Imho that has more transparency, then just sending a mail to a random address, containing the app identifier I'm working on.
